cmd/client/main.go

```diff
@@ -212,15 +208,12 @@ func run(flags *flags) {
 		)
 	// For tests that depend on trailers, we only run them for HTTP2, since the HTTP3 client
 	// does not yet have trailers support https://github.com/lucas-clemente/quic-go/issues/2266
-	case connectH3, connectGRPCH3, connectGRPCWebH3:
+	// connectGRPCH3 and connectGRPCWebH3 have both been disabled since we are now strictly
+	// requiring `grpc-status` headers to be set on response, which requires trailer support.
+	// Once trailer support is available, they will be renabled.
```

Comment:
Do we need to disable gRPC-web? The protocol doesn't use HTTP trailers. It does have a concept of trailers, but they are encoded in the response body. 
As far as I see, the lack of trailer support in the H3 implementation is only an issue for gRPC.